dashboard/api_test.py

```diff
@@ -145,6 +146,81 @@ def test_format_run_no_run(self):
             api.format_courserun_for_dashboard(None, api.CourseStatus.PASSED)
         )
 
+    def test_price_in_course_run(self):
+        """Assert that price appears in course run for offered and upgrade."""
+        crun = self.create_run()
+        CoursePriceFactory.create(course_run=crun, is_valid=True, price=50)
+
+        self.assertEqual(
+            api.format_courserun_for_dashboard(crun, api.CourseStatus.OFFERED),
+            {
+                'title': crun.title,
+                'status': api.CourseStatus.OFFERED,
+                'id': crun.pk,
+                'course_id': crun.edx_course_key,
+                'enrollment_start_date': crun.enrollment_start,
+                'fuzzy_enrollment_start_date': crun.fuzzy_enrollment_start_date,
+                'position': 1,
+                'price': 50.0
+            }
+        )
+        self.assertEqual(
+            api.format_courserun_for_dashboard(crun, api.CourseStatus.UPGRADE),
+            {
+                'title': crun.title,
+                'status': api.CourseStatus.UPGRADE,
+                'id': crun.pk,
+                'course_id': crun.edx_course_key,
+                'position': 1,
+                'price': 50.0
+            }
+        )
+        self.assertNotEqual(
```

Comment:
`assertNotEqual` is not very useful here since any number of things can cause two dictionaries to be different. Can you change the below four assertions to assert that `price` does not exist in the returned dictionary?